zerver/openapi/zulip.yaml

```diff
@@ -5606,6 +5606,14 @@ paths:
             in UTC seconds.
           required: true
           example: 3165826990
+        - name: read_by_sender
+          in: query
+          schema:
+            type: boolean
+          description: |
+            Whether the message should be initially marked read by its
+            sender.
```

Comment:
These need **Changes** entries.

Reply:
They’re there, in `api_docs/changelog.md`.

Reply:
Yes but it's a double-entry system -- we always detail API changes in both the global changelog and also locally in the location of the individual endpoint/parameter, so you don't have to remember to check the changelog to just write correct mobile client code for a given endpoint.

Reply:
Oh right, fixed.